src/GNUmakefile

```diff
@@ -213,7 +216,7 @@ extension/python/python_extension.o: python_extension.py cli/cli.o
 
 .SECONDARY: proot_wrap.c proot.py
 proot_wrap.c proot.py: extension/python/proot.i
-	$$($$(quiet)SWIG) -python -outcurrentdir -I$$(VPATH) $$(VPATH)/extension/python/proot.i
+	$$($$(quiet)SWIG) -python -py3 -outcurrentdir -I$$(VPATH) $$(VPATH)/extension/python/proot.i
```

Comment:
Can this be put in a variable? Perhaps `SWIG_OPT="-python -py3"` or something like that. Then we could override the version.